packages/manager/apps/pci-workflow/src/api/hooks/useInstances.ts

```diff
@@ -0,0 +1,101 @@
+import { useMemo } from 'react';
+import { applyFilters, Filter } from '@ovh-ux/manager-core-api';
+import { getFlavor, TInstance, useInstances } from '@ovh-ux/manager-pci-common';
+import { useQueries, UseQueryResult } from '@tanstack/react-query';
+import { useTranslatedMicroRegions } from '@ovh-ux/manager-react-components';
+import { sortResults, TInstanceOptions } from '@/api/data/instance';
+import { paginateResults } from '@/helpers';
+import { TWorkflowInstance } from '@/types';
+
+export const getStatusGroup = (status: string) => {
+  if (
+    [
+      'BUILDING',
```

Comment:
might want to declare an enum for that

Reply:
typescript string enums are very verbose, also we have to check the status at runtime and not at compilation time so i'm not sure about it

could you provide an example of what you consider a better way of handling this case ? thanks